src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -28,15 +28,15 @@ <h5 class="d-inline"><strong> Sort By: </strong></h5>
       <div class="btn-group" data-toggle="buttons">
         <button class="btn btn-light" [disabled]="isSelectedForSorting(SortBy.COURSE_ID)" (click)="sortCoursesBy(SortBy.COURSE_ID)">Course ID</button>
         <button class="btn btn-light" [disabled]="isSelectedForSorting(SortBy.COURSE_NAME)" (click)="sortCoursesBy(SortBy.COURSE_NAME)">Course Name</button>
-        <button class="btn btn-light" [disabled]="isSelectedForSorting(SortBy.CREATION_DATE)" (click)="sortCoursesBy(SortBy.CREATION_DATE)">Creation date</button>
+        <button class="btn btn-light" [disabled]="isSelectedForSorting(SortBy.COURSE_CREATION_DATE)" (click)="sortCoursesBy(SortBy.COURSE_CREATION_DATE)">Creation Date</button>
       </div>
     </div>
   </div>
 </div>
 
 <div *ngIf="courseTabModels.length > 0; else noCourseMessage">
   <div class="card margin-bottom-15px" *ngFor="let courseTabModel of courseTabModels; let idx = index">
-    <div class="card-header bg-primary text-white course-tab-header" (click)="courseTabModel.isTabExpanded = !courseTabModel.isTabExpanded">
+    <div class="card-header bg-primary text-white course-tab-header" (click)="courseTabModel.isTabExpanded = !courseTabModel.isTabExpanded; this.loadFeedbackSessions(courseTabModel);">
```

Comment:
Let's shift the logic of this line into the `loadFeedbackSessions` method of the TS file instead:
```
courseTabModel.isTabExpanded = !courseTabModel.isTabExpanded
```

Also I believe `loadFeedbackSessions(courseTabModel)` is sufficient, don't need `this.loadFeedbackSessions(courseTabModel)`